### ISSUE TITLE
Wrong blockchain explorer URL in withdrawal history for destination address

### ISSUE THREAD

**Issue by allenmillen** [openware/peatio#579]
When withdrawal is submitted, wrong URL is shown in withdrawals history (it is pointing to tx). It should be pointing to address instead

**Comment by yivo** [comment]
This is known issue. Configuration problem.

**Comment by yivo** [comment]
Fixed public/templates/funds/_deposit_coin_history.html
```
<br />
<h2 class="panel-title">{{ "funds.deposit_history.title" | t }}</h2>
<table class="table" ng-controller="DepositHistoryController as dhCtrl">
  <tr>
    <th>{{ "funds.deposit_history.time" | t }}</th>
    <th>{{ "funds.deposit_history.txid" | t }}</th>
    <th>{{ "funds.deposit_history.amount" | t }}</th>
    <th>{{ "funds.deposit_history.confirmations" | t }}</th>
    <th>{{ "funds.deposit_history.state_and_action" | t }}</th>
  </tr>
  <tr ng-class="{ 'new-row': dhCtrl.newRecord(deposit) }" ng-repeat="deposit in dhCtrl.deposits | orderBy:predicate">
    <td>{{deposit.created_at | date:'yyyy-MM-dd HH:mm'}}</td>
    <td>{{deposit.txid}}</td>
    <td>{{deposit.amount}}</td>
    <td>{{deposit.confirmations}}</td>
    <td>{{"funds.deposit_history." + deposit.aasm_state | t}}</td>
  </tr>
  <tr ng-show="dhCtrl.noDeposit()"><td colspan="6"><span class="help-block text-center">{{ "funds.deposit_history.no_data" | t }}</span></td></tr>
</table>

```

**Comment by frezeq00** [comment]
bittrex opensource: https://github.com/pkdcryptos/frezeq0_bittrex-opensource

**Comment by yivo** [comment]
Closed via #616.
